adhocracy-plus/assets/js/app.js

```diff
@@ -19,9 +19,9 @@ import {
   widget as ReactWidget
 } from 'adhocracy4'
 
-import * as ReactDocuments from '../../../apps/documents/assets/react_documents.jsx'
-import * as ReactInteractiveEvents from 'a4_candy_interactive_events'
-import * as ReactLanguageChoice from '../../../apps/organisations/assets/react_language_choice.jsx'
+import { renderDocumentManagement } from 'react_documents'
```

Comment:
is there any advantage using the webpack resolve things over just directly importing it? Needing the webpack resolve stuff feels a bit heavy / more to maintain than just having the `../../../apps/` import

Reply:
not really thought about performance.
I thought it is less to maintain if we would for instance use those apps in different projects, but having them each in different folder structures. And we only set them in one spot (webpack config), rather than having different app.js files. But i might idealize it here...

personally i do not really like `../../` maybe i can try resolve the path into `apps/` and then we have something like:
`apps/organisations/assets/react_language_choice.jsx` for instance?

Reply:
it's not about performance, I just wasn't sure if having code in different files (e.g. code for imports in webpack config) makes it easier or harder to maintain. I also don't know it, so if you think we should stick to your solution I'm absolutely fine with that :) 
 > I thought it is less to maintain if we would for instance use those apps in different projects, but having them each in different folder structures. And we only set them in one spot (webpack config), rather than having different app.js files. But i might idealize it here...

I didn't really take that into account so maybe you are right

Reply:
@goapunk i decided to go without webpack resolving, because apart from maintainance that might be more complicated indeed, this way i can keep `.eslint` identical in a4 and aplus. So latest commit has it already.